docs/Validation.md

```diff
@@ -259,6 +259,85 @@ export const ProductEdit = () => (
 
 **Tip**: The custom validator function can return a promise, e.g. to use server-side validation. See next section for details.
 
+## Per Input Validation: useUnique
```

Comment:
I think this hook deserves its own page (and link in the Reference documentation). It's a good idea to link to it from the Validation doc, though (in [the list of built-in validators](https://marmelab.com/react-admin/Validation.html#per-input-validation-built-in-field-validators))